src/use-css.hook.js

```diff
@@ -3,7 +3,7 @@ import {Scoped} from './scoped.component.js'
 import {styleTags, incrementCounter, transformCss} from './style-element-utils.js'
 
 export function useCss(css, overrideNamespace) {
-  const isPostCss = Boolean(css && css.id && css.styles)
+  const isPostCss = Boolean(css && css.id)
```

Comment:
maybe `typeof css.styles === 'string'` would be better? We need to ensure that it's a string because we are [setting css.styles as the textContent of a dom node](https://github.com/CanopyTax/kremling/blob/bde0d7197e1c1f98fdfe3bce8b3341843bae0423/src/use-css.hook.js#L48)

Reply:
yeah... I was thinking that implicitly checking for `css.id` would tell us that it's coming from the `kremling-loader` - if either `css.id` or `css.styles` is missing in that case then it'll still break because `css` is still an object

I'm okay to check each property, but I'm thinking we need a fallback if a property is missing